BAZProject/Trending/TrendingViewModel.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  TrendingViewModel.swift
+//  BAZProject
+//
+//  Created by jehernandezg on 30/03/23.
+//
+
+import UIKit
+import Foundation
+
+/// Protocol that is responsible for having the logic of movies to be called in the view
+protocol MovieListProtocol {
+    var movies: [ListMovieProtocol]? { get set }
+    func getDataMovies(_ Completion: @escaping () -> Void)
+    func getRowCount() -> Int
+    func getTitle(index: Int) -> String?
+    func getPosterPath(index: Int) -> String?
+}
+
+/// Structure that conforms to the MovieListProtocol protocol
+class TrendingViewModel: MovieListProtocol {
+    
+    var movies: [ListMovieProtocol]?
+    
+    func getDataMovies(_ Completion: @escaping () -> Void) {
```

Comment:
los parameter names usan la convención lowerCamelCase
```suggestion
    func getDataMovies(_ completion: @escaping () -> Void) {
```